examples/amp-story/shop-for-stuff.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story"
+        src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async src="https://cdn.ampproject.org/v0/amp-viewer-integration-0.1.js"></script>
+    <title>My Story</title>
+    <meta name="viewport"
+          content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link rel="canonical" href="helloworld.html">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        font-family: 'Roboto', sans-serif;
+      }
+      amp-story-page {
+        background: white;
+      }
+      h1 {
+        font-size: 2.875em;
+        font-weight: normal;
+        line-height: 1.174;
+        text-transform: uppercase;
+      }
+      .pinkBackground {
+        background: rgba(255, 0, 0, 0.3);
+      }
+      .top {
+        position: absolute;
+        left: 0;
+        top: 0;
+        width: 20px;
+        height: 20px;
+        background: green;
+      }
+      .bottom {
+        position: absolute;
+        right: 0;
+        bottom: 0;
+        width: 20px;
+        height: 20px;
+        background: blue;
+      }
+    </style>
+  </head>
+
+  <body>
+    <amp-story standalone>
+
+      <amp-story-page id="cover" auto-advance-after="5s">
+        <amp-story-grid-layer class="pinkBackground" template="vertical">
+          <h1>Shop For Stuff</h1>
+          <p>Add items from this story into your cart for purchase.</p>
+        </amp-story-grid-layer>
+      </amp-story-page>
+
+      <amp-story-page id="page-1" auto-advance-after="5s">
+        <amp-story-grid-layer class="pinkBackground" template="vertical">
+          <h1>Laptop</h1>
+          <p>Buy this MacBook Air</p>
+          <amp-img src="https://picsum.photos/id/0/5616/3744"
+              width="300"
+              height="170">
+          </amp-img>
+          <a player-action='{"player-action-id":"shopForStuffAddToCart", "productId":"macbook"}' data-expected-host-domains="localhost www.google.com" data-fallback-url="https://www.google.com/">
```

Comment:
It's possible to parse data attributes as json. 
This way the template can be more readable and easier for publishers to set up.

You can do this by converting `player-action` to a data attribute by prepending `data-` so:
`data-player-action`

[demo JSfiddle](http://jsfiddle.net/s2wn4adg/1/)